packages/amazon/src/serverGroup/details/sections/LaunchConfigDetailsSection.tsx

```diff
@@ -58,8 +58,10 @@ export class LaunchConfigDetailsSection extends React.Component<
             <dt>IAM Profile</dt>
             <dd>{launchConfig.iamInstanceProfile}</dd>
 
-            <dt>Instance Monitoring</dt>
-            <dd>{launchConfig.instanceMonitoring.enabled ? 'enabled' : 'disabled'}</dd>
+            {launchConfig.instanceMonitoring && <dt>Instance Monitoring</dt>}
+            {launchConfig.instanceMonitoring && (
```

Comment:
Can merge the two instanceMonitoring checks and render the title and body in one block?